enterprise/management/commands/seed_enterprise_devstack_data.py

```diff
@@ -0,0 +1,252 @@
+            name=customer_name,
+            site_id=site.id,
+            country='US',
+            slug=slugify(customer_name),
+            enable_data_sharing_consent=True,
+            enable_portal_code_management_screen=True,
+            enable_portal_reporting_config_screen=True,
+        )
+        return enterprise_customer
+
+    def _create_catalog_for_enterprise(self, enterprise_customer):
+        """ Gets or creates a catalog for an EnterpriseCustomer """
+        catalog, __ = EnterpriseCustomerCatalog.objects.get_or_create(
+            title='All Course Runs',
+            enterprise_customer=enterprise_customer,
+            content_filter=CATALOG_CONTENT_FILTER,
+        )
+        return catalog
+
+    def _create_enterprise_data_api_group(self):
+        """ Ensures the `ENTERPRISE_DATA_API_ACCESS_GROUP` is created """
+        Group.objects.get_or_create(name=ENTERPRISE_DATA_API_ACCESS_GROUP)
+
+    def _create_enterprise_enrollment_api_group(self):
+        """ Ensures the `ENTERPRISE_ENROLLMENT_API_ACCESS_GROUP` is created """
+        Group.objects.get_or_create(name=ENTERPRISE_ENROLLMENT_API_ACCESS_GROUP)
+
+    def _create_enterprise_user(self, username, role):
+        """ 
+        Creates a new user with the specified `username` and `role` (e.g., 
+        'enterprise_learner'). The newly created user is added to the 
+        appropriate Django groups (e.g., data api access) and creates 
+        system-wide and feature role assignments.
+        """
+        valid_roles = [
+            ENTERPRISE_LEARNER_ROLE,
+            ENTERPRISE_ADMIN_ROLE,
+            ENTERPRISE_OPERATOR_ROLE,
+        ]
+        if role in valid_roles:
+            is_staff = True if role == ENTERPRISE_OPERATOR_ROLE else False
+            try:
+                user = User.objects.create_user(
+                    email='{username}@example.com'.format(username=username),
+                    username=username,
+                    password='edx',
+                    is_staff=is_staff,
+                )
+            except IntegrityError:
+                # If a user we attempt to create in this method already exists but with
+                # slightly different parameters, we will attempt to use the existing user.
+                user = User.objects.get(username=username)
+            if user:
+                self._add_user_to_groups(user=user, role=role)
+                self._create_system_wide_role_assignment(user=user, role=role)
+                self._create_feature_role_assignments(user=user, role=role)
+                return {
+                    "user": user,
+                    "role": role,
+                }
+        else:
+            LOGGER.warning('\nUser not created. Role %s not recognized.', role)
+        return None
+
+    def _add_user_to_groups(self, user, role):
+        """ Adds a user with a given role to the appropriate groups """
+        if role != ENTERPRISE_LEARNER_ROLE:
```

Comment:
Alternative syntax suggestion, in case you wanted to save indentation space for the following lines:

```
if role == ENTERPRISE_LEARNER_ROLE:
    return
data_api_group = Group.objects.get(name=ENTERPRISE_DATA_API_ACCESS_GROUP)
data_api_group.user_set.add(user)
enrollment_api_group = Group.objects.get(name=ENTERPRISE_ENROLLMENT_API_ACCESS_GROUP)
enrollment_api_group.user_set.add(user)
```